Nasal/c172p.nas

```diff
@@ -191,6 +191,68 @@ var return_fuel_sample = func(index) {
 };
 
 ##########################################
+# Preflight control surface check: left aileron
+##########################################
+var control_surface_check_left_aileron = func {
+    var auto_coordination = getprop("/controls/flight/auto-coordination");
+    setprop("/controls/flight/auto-coordination", "false");
+    interpolate("/controls/flight/aileron", 1.0, 0.3);
```

Comment:
According to the [documentation](http://wiki.flightgear.org/Nasal_library#interpolate.28.29) you should be able to replace this with:

```javascript
interpolate("/controls/flight/aileron", 1.0, 0.3, -1.0, 0.6, 0.0, 0.3);
```